src/components/Navbar/Navbar.jsx

```diff
@@ -28,14 +32,38 @@ export default function Navbar() {
           {menuItems.map((item) => {
             return (
               <li key={item.name}>
-                <a href={item.href}>{item.name}</a>
+                <a href={item.href}>
+                  <FormattedMessage id={item.id} defaultMessage={item.name} />
+                </a>
               </li>
             );
           })}
         </Menu>
+        <LanguagesContainer>
+          {languages?.map((item) => {
```

Comment:
optional chaining (?) jest tutaj niepotrzebny, bo lista będzie zawsze miała jakąś wartość - pierdoła, ale dodanie optional chainingu zwiększa rozmiar zbudowanej paczki i przy kilkuset takich nadrmiarowych '?' może ważyć więcej - warto o tym wiedzieć jakby takie pytanie padło na rozmowie technicznej - u Ciebie to bez znaczenia, jeden nie zaszkodzi aplikacji :D